app/views/schools/partnerships/index.html.erb

```diff
@@ -1,14 +1,20 @@
+<% if @partnership %>
+<% content_for :title, "Signed up with provider" %>
+<% else %>
+<% content_for :title, "Confirm with provider" %>
+<% end %>
+
 <% content_for :before_content, govuk_back_link(
   text: "Back",
   href: schools_cohort_path(@cohort.start_year))
 %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-m"><%= @cohort.start_year %></span>
-    <h1 class="govuk-heading-l">Sign up with a training provider</h1>
+    <span class="govuk-caption-l"><%= @cohort.start_year %> cohort</span>
     <% if @partnership %>
-    <h2 class="govuk-heading-m">Partnership details</h2>
```

Comment:
I think it might be nice to split these into partials. What do you think?

Reply:
I've done it